compose/ts-translation-service.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.4'
+
+services:
+  ts-translation-service:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: "hmctspublic.azurecr.io/ts/translation-service:latest"
+    container_name: translation-service
+    environment:
+      TS_TRANSLATION_SERVICE_DB_HOST: "${CCD_POSTGRES_11:-ccd-shared-database}"
+      TS_TRANSLATION_SERVICE_DB_PORT: 5432
+      TS_TRANSLATION_SERVICE_DB_NAME: ts_translation_service
+      TS_TRANSLATION_SERVICE_DB_USERNAME: ${DB_USERNAME}
+      TS_TRANSLATION_SERVICE_DB_PASSWORD: ${DB_PASSWORD}
+      IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_OIDC_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      S2S_URL: http://service-auth-provider-api:8080
```

Comment:
For consistancy with other services can we add:

* `TS_TRANSLATION_SERVICE_S2S_AUTHORISED_SERVICES` (see hmcts/ts-translation-service#6)
* `REFORM_SERVICE_NAME: ts-translation-service`
* `REFORM_TEAM: ccd`
* `REFORM_ENVIRONMENT: local`
* `APPINSIGHTS_INSTRUMENTATIONKEY: key`